build.ps1

```diff
@@ -68,15 +68,16 @@ Function BuildProject($path, $message) {
     }
 }
 
-Function RunApp($path, $message, $args) {
+Function RunApp($path, $message, $params) {
 
-    $label = "RUN: --- $message $configuration ($args)---"
+    $label = "RUN: --- $message $configuration ($params)---"
 
     Write-Host
     Write-Host -ForegroundColor Cyan $label
     cd (Join-Path $rootDir $path)
 
-    & dotnet run $args
+    $runCommand = "dotnet run -- $params"
+    Invoke-Expression $runCommand

```

Comment:
The native dotnet run command when invoked using `&` was showing this weird behavior, invoking it using `Invoke-Expression` works just fine.